### PR TITLE
fix:[PLG-560] Cherrypick - CS and R7 Admin policy severity and MoreInfo link

### PR DESCRIPTION
**Pull request by ershad-paladin**
## Description

- get the actual severity of policy from policy params to fetch vulnerabilities, as the admin can change the policy's severity
- add resolutionUrl to the CS policies so that MoreInfo link on the policy details page point to our policy docs.
- This is a cherry pick of https://github.com/PaladinCloud/CE/pull/2217

## Type of change

**Please delete options that are not relevant.**

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Chore (no code changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [ ] Change the severity of a CS policy in admin policy
- [ ] run the respective policy rule job
- [ ] Violation must be created base on the actual policy independent of the policy's severity
- [ ] click on MoreInfo link on the policy details page and it should open the paladin aws policy docs

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] My commit message/PR follows the contribution guidelines of this project
- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by rnithinpaladin — APPROVED** [review]

File: installer/resources/pacbot_app/files/DB_Policy.sql [hunk 1]
```diff
@@ -3383,34 +3383,47 @@ update  cf_PolicyTable set policyDesc = 'Using a single IAM access key for IAM u
...
 INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `defaultVal`, `isEdit`, `isMandatory`, `encrypt`, `displayName`, `description`) VALUES ('ec2_r7_vulnerability_critical','vulnerability_index','rapid7_vulnerability','','false','false','false','','');
 INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `defaultVal`, `isEdit`, `isMandatory`, `encrypt`, `displayName`, `description`) VALUES ('ec2_r7_vulnerability_critical','asset_lookup_key','instanceId','','false','false','false','','');
 INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `defaultVal`, `isEdit`, `isMandatory`, `encrypt`, `displayName`, `description`) VALUES ('ec2_r7_vulnerability_critical','policyKey','check-vm-vulnerabilities-scanned-by-plugin-grouped','','false','false','false','','');
+INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `defaultVal`, `isEdit`, `isMandatory`, `encrypt`, `displayName`, `description`) VALUES ('ec2_r7_vulnerability_critical','severityToCheck','critical','','false','false','false','','');
```
Comment (plyubich) [inline installer/resources/pacbot_app/files/DB_Policy.sql:3390]: can you rename this to simply severityMatchCriteria

File: jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/awsrules/ec2/AssetTypeGroupedVulnerabilitiesRule.java [hunk 2]
```diff
@@ -82,7 +83,7 @@ public PolicyResult execute(Map<String, String> ruleParam, Map<String, String> r
             String instanceId = StringUtils.trim(resourceAttributes.get(PacmanRuleConstants.RESOURCE_ID));
             List<JsonObject> vulnerabilityInfoList = new ArrayList<>();
             try {
-                vulnerabilityInfoList = PacmanUtils.matchAssetAgainstSourceVulnIndex(instanceId, vulnerabilitiesEndpoint, vulnAssetLookupKey, null);
+                vulnerabilityInfoList = PacmanUtils.matchAssetAgainstSourceVulnIndex(instanceId, vulnerabilitiesEndpoint, vulnAssetLookupKey, severityToCheck);
```
Comment (plyubich) [inline jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/awsrules/ec2/AssetTypeGroupedVulnerabilitiesRule.java:86]: please rename to severityMatchCriteria

File: jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/constants/PacmanRuleConstants.java [hunk 3]
```diff
@@ -113,6 +113,7 @@ private PacmanRuleConstants() {
     public static final String FOUND = "Found";
     public static final String NOTFOUND = "Not Found";
     public static final String SEVERITY = "severity";
+    public static final String SEVERITY_TO_CHECK = "severityToCheck";
```
Comment (plyubich) [inline jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/constants/PacmanRuleConstants.java:116]: severityMatchCriteria

**Review by plyubich — COMMENTED** [review]
@ershad-paladin please follow up with a new PR for change to severityMatchCriteria

**Pull request merged**